haskell-src/Concordium/Types/Queries.hs

```diff
@@ -119,7 +119,17 @@ data ConsensusStatus = ConsensusStatus
       -- Initially this is equal to 'csGenesisBlock'.
       csCurrentEraGenesisBlock :: !BlockHash,
       -- |Time when the current era started.
-      csCurrentEraGenesisTime :: !UTCTime
+      csCurrentEraGenesisTime :: !UTCTime,
+      -- |The current duration to wait before a round times out.
+      csCurrentTimeoutDuration :: !(Maybe Duration),
+      -- |The current round.
+      csCurrentRound :: !(Maybe Word64),
+      -- |The current epoch.
+      csCurrentEpoch :: !(Maybe Epoch),
+      -- |The trigger block time of the seedstate of the last finalized block. The first block in
+      -- the epoch with timestamp at least this is considered to be the trigger block for the epoch
+      -- transition.
```

Comment:
Can you provide documentation explaining this is present if and only if protocol version is >= 6